docs/website/mmp-errors.md

```diff
@@ -236,6 +236,12 @@ Please file an [issue](https://github.com/xamarin/xamarin-macios/issues/new)
 along with a complete build log so that we can investigate what went wrong and
 possibly enable more scenarios in the future.
 
+### <a name="MM2107"/>MM2107: It's not safe to remove the dynamic registrar because {reasons}
+
+The linker reports this warning when the developer requests removal of the
+dynamic registrar (by passing `--optimize:remove-dynamic-registrar` to
+mmp), but the linker determines that it's not safe to do so.
+
```

Comment:
missing action ;-) e.g.

The default optimization option was overridden in your project. Remove it to get rid of the warning at build time.
Note: By default this option will be automatically enabled whenever it's possible/safe to apply.